http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleDataprocDAO.scala

```diff
@@ -231,7 +230,7 @@ class HttpGoogleDataprocDAO(appName: String,
     // Create a NodeInitializationAction, which specifies the executable to run on a node.
     // This executable is our init-actions.sh, which will stand up our jupyter server and proxy.
     val initActions = config.initScripts.map { script =>
-      new NodeInitializationAction().setExecutableFile(script.toUri).setExecutionTimeout(finiteDurationToGoogleDuration(defaultExecutionTimeout))
+      new NodeInitializationAction().setExecutableFile(script.toUri)
```

Comment:
If this is not set does it default to infinite, or is there a dataproc default timeout?